hyrax.gemspec

```diff
@@ -25,34 +25,34 @@ EOF
   spec.version       = Hyrax::VERSION
   spec.license       = 'Apache-2.0'
 
-  spec.add_dependency 'hydra-head', '>= 10.4.0'
+  spec.add_dependency 'hydra-head', '~> 10.4', '>= 10.4.0'
   spec.add_dependency 'hydra-editor', '~> 3.2'
   spec.add_dependency 'hydra-works', '~> 0.16'
   spec.add_dependency 'hydra-derivatives', '~> 3.3'
-  spec.add_dependency 'browse-everything', '>= 0.10.5'
+  spec.add_dependency 'browse-everything', '~> 0.10', '>= 0.10.5'
   spec.add_dependency 'blacklight', '~> 6.9'
   spec.add_dependency 'blacklight-gallery', '~> 0.7'
   spec.add_dependency 'tinymce-rails', '~> 4.1'
   spec.add_dependency 'tinymce-rails-imageupload', '~> 4.0.17.beta'
   spec.add_dependency 'font-awesome-rails', '~> 4.2'
-  spec.add_dependency 'select2-rails', '~> 3.5.9'
-  spec.add_dependency 'json-schema' # for Arkivio
+  spec.add_dependency 'select2-rails', '~> 3.5', '>= 3.5.9'
+  spec.add_dependency 'json-schema', '~> 0' # for Arkivio
```

Comment:
This line appears to cause bundler to pick up version 0.9.12 of json-schema. I'm seeing the most recent version (2.8.0) used before this change. That's why Travis is failing, methinks. This makes me wonder if the other `~> 0` declarations in the gemspec are potentially problematic as well, and I'm not sure how `~> 0` is any clearer or more tightly specified than no version at all.